fake-snippets-api/routes/api/orders/create.ts

```diff
@@ -198,11 +197,6 @@ export default withRouteSpec({
     newOrder.has_error = true
   }
 
-  if (newOrder.has_error) {
-    return ctx.json({
-      order: newOrder,
-    })
-  }
```

Comment:
If the order fails to create, we just return { error }

Reply:
For simulated errors, you have to poll for them just like the real api, they dont happen instantly

Reply:
Ok i see whats going on. I think we still add the order to the fb if there’s an error, the real behavior is the order gets added, then asynchronously in the job queue it gets the error attached